packages/adminrouter/extra/src/README.md

```diff
@@ -31,6 +31,229 @@ Use `make check-api-docs` to validate that the YAML and HTML files are up to dat
 ## Ports summary
 <img src="admin-router-table.png" alt="" width="100%" align="middle">
 
+## Repository structure
+
+There are two Admin Router "flavours" residing in DC/OS repos:
+  * https://github.com/dcos/dcos/tree/master/packages/adminrouter - Opensource version or `Open` in short
```

Comment:
❓ We don't use a Markdown linter and I'd rather not be a human linter but... my linter complains if you put URLs in which aren't surrounded by angle brackets.